packages/w3up-client/test/mocks/service.js

```diff
@@ -0,0 +1,43 @@
+import * as Client from '@ucanto/client'
+import * as Server from '@ucanto/server'
+import * as CAR from '@ucanto/transport/car'
+
+import * as AccessCaps from '@web3-storage/capabilities'
+
+/**
+ * Mocked Gateway/Content Serve service
+ */
+export function getMockService() {
+  return {
+    access: {
+      delegate: Server.provide(
+        AccessCaps.Access.delegate,
+        async ({ capability, invocation }) => {
+          return {
+            ok: {},
+          }
+        }
+      ),
+    },
+  }
+}
+
+/**
+ * @param {any} service
+ * @param {any} id
```

Comment:
**suggestion:** Let's give these real types.

Reply:
I am not sure if we need it in this case. This is like a generic function to instantiate connections for any type of service with any identifier.